tests/browser_test.go

```diff
@@ -217,6 +217,9 @@ func TestBrowserLogIterationID(t *testing.T) {
 }
 
 func TestMultiBrowserPanic(t *testing.T) {
+	// TODO: This test never works on CI, fix it.
+	t.Skip("skipping, fix this")
```

Comment:
I wonder if we should use an ENV var, set by us in CI, as a flag in order to skip this test or not. In that way at least we will always run this test when testing locally. Do you think that is worth it?

Reply:
Makes sense; I was thinking about the same, too. Especially for the `disable-gpu` one. We can do it whether we're running on Github by checking the `GITHUB_ACTIONS` environment variable.